data/database.py

```diff
@@ -427,11 +427,16 @@ def _db_from_url(
     allow_pooling=True,
     allow_retry=True,
     is_read_replica=False,
+    builder_host_override=None,
 ):
     parsed_url = make_url(url)
 
     if parsed_url.host:
         db_kwargs["host"] = parsed_url.host
+        service_name = os.getenv("SUPERVISOR_PROCESS_NAME")
```

Comment:
How is this `SUPERVISOR_PROCESS_NAME` set?

Reply:
seems to be set by [supervisord](http://supervisord.org/subprocess.html#subprocess-environment)

Reply:
Ah, gotcha, that makes sense :-)